demo/AsyncApp.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+/* eslint-disable no-console, react/no-multi-comp */
+import React from 'react';
+
+import {
+  CardElement,
+} from '../src/index';
+
+const createOptions = (fontSize: string) => {
+  return {
+    style: {
+      base: {
+        fontSize,
+        color: '#424770',
+        letterSpacing: '0.025em',
+        fontFamily: 'Source Code Pro, monospace',
+        '::placeholder': {
+          color: '#aab7c4',
+        },
+      },
+      invalid: {
+        color: '#9e2146',
+      },
+    },
+  };
+};
+
+// We load Stripe immediately, so we'll add some delay.
```

Comment:
We could add `We load Stripe immediately, so we'll add some delay to simulate asynchronous loading.`